src/lib/data.ts

```diff
@@ -43,9 +46,39 @@ export function setWithoutEmit(key: string, value: any | undefined): any {
     return value;
 }
 
-export function set(key: string, value: any | undefined): any {
+export function set(key: string, value: any | undefined, persist: boolean = false): any {
```

Comment:
nit: I wonder if there is a clearer parameter name here than `persist`.  This concern based on my understanding that localStorage is also "persistent" (outlasting the lifetime of the session). Perhaps `persist_remotely`?

Reply:
Yeah naming eh!   I changed that a few times already, happy to change it again.